versal.mk

```diff
@@ -0,0 +1,161 @@
+################################################################################
+# Following variables defines how the NS_USER (Non Secure User - Client
+# Application), NS_KERNEL (Non Secure Kernel), S_KERNEL (Secure Kernel) and
+# S_USER (Secure User - TA) are compiled
+################################################################################
+override COMPILE_NS_USER   := 64
+override COMPILE_NS_KERNEL := 64
+override COMPILE_S_USER    := 64
+override COMPILE_S_KERNEL  := 64
+
+PLATFORM = versal-vck190
+OPTEE_OS_PLATFORM = versal
+
+################################################################################
+# Paths to git projects and various binaries
+################################################################################
+TF_A_PATH		?= $(ROOT)/arm-trusted-firmware
+U-BOOT_PATH		?= $(ROOT)/u-boot-xlnx
+BOOTGEN_PATH		?= $(ROOT)/bootgen
+LINUX_PATH		?= $(ROOT)/linux-xlnx
```

Comment:
Minor: does the `-xlnx` suffixes really help anything? Other platforms don't have them.

Reply:
yes you are right, it doesnt. will remove. let me update

Reply:
going to retest everything and then send the manifest. 

BTW - just a side note-  the qemu target is no longer in the zynqmp.mk and the  docs refer to it 
https://optee.readthedocs.io/en/latest/building/devices/zynqmp.html
am I missing something?